rocketpy/stochastic/stochastic_model.py

```diff
@@ -63,9 +66,24 @@ def __init__(self, obj, **kwargs):
 
         self.obj = obj
         self.last_rnd_dict = {}
+        self.__stochastic_dict = kwargs
+        self._set_stochastic()
```

Comment:
```suggestion
        self._set_stochastic(seed)
```

Reply:
I wonder whether you made a mistake

Reply:
It is a mistake, nice catch.

Reply:
Solved by [abe5747](https://github.com/RocketPy-Team/RocketPy/pull/649/commits/abe574774fa7bc8e16b5fcc62ab4c9e4dfc40afa).